### PR TITLE
test(driver-adapters): vercel nextjs edge middleware

### PR DESCRIPTION
**Pull request by millsp**

### CODE REVIEW

File: driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  webpack(config) {
+    config.experiments ??= {}
+
+    config.experiments.asyncWebAssembly = true
```
Comment (janpio) [inline driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js:8]: Always needed for our users?

File: driver-adapters/planetscale-vercel-nextjs-edgemw/prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,18 @@
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+  engineType      = "wasm"
+}
+
+datasource db {
+  provider = "mysql"
+  // TODO: needs to be changed to a fake url to ensure the driver adapter is used
+  url      = env("DRIVER_ADAPTERS_PLANETSCALE_VERCEL_NEXTJS_EDGEMW_DATABASE_URL")
```
Comment (Jolg42) [inline driver-adapters/planetscale-vercel-nextjs-edgemw/prisma/schema.prisma:10]: TODO?

File: driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js (hunk 1 above)
Comment (Jolg42) [inline driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js:8]:
```suggestion
    // https://webpack.js.org/configuration/experiments/#experiments
    config.experiments.asyncWebAssembly = true
```
It's nice to link the docs I think

**Review by Jolg42 — APPROVED** [review]

File: driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js (hunk 1 above)
Comment (millsp) [reply driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js:8]: Yes, unfortunately. Without it, you get an error that tells you to add it.

**Pull request merged**